### DISCUSSION TITLE
Update many dynamic inputs fields

### DISCUSSION THREAD

**Discussion by aspyropoulos** [alpinejs/alpine#2202] [1. Help] [answered]
How can I update many dynamic input fields in Alpinejs. 
I have created [this fiddle](https://jsfiddle.net/n76cv3ut/26/). How can I make it work with AlpineJS assuming 
1 to N inputs.

**Comment by SimoTod** [comment, marked as answer]
You can combine x-model and x-effect: https://codepen.io/SimoTod/pen/GRvRrjq
Note, fields update on input, if you want to really do it on change, change `x-model="foo"` to `x-model.lazy="foo"`
